openassessment/xblock/team_mixin.py

```diff
@@ -99,6 +104,41 @@ def valid_access_to_team_assessment(self):
         """
         return self.is_course_staff or self.in_studio_preview or self.has_team()
 
+    def add_team_submission_context(
```

Comment:
I thought 'get_team_info()' below added that information to context?

Reply:
get_team_info adds information to the context based on team memberships, this specifically is adding that info from a submission

Reply:
but uh, wait, yeah, maybe it should? If there's no submission we call get_team_info which adds additional info to the context ....

Reply:
okay ignore these comments I'm gonna think abut this a bit more